nuxt.config.js

```diff
@@ -11,28 +11,41 @@ export default {
       { hid: "description", name: "description", content: "" },
       { name: "format-detection", content: "telephone=no" },
     ],
-    link: [{ rel: "icon", type: "image/x-icon", href: "/img/12-labours-logo-black.png" }],
-
+    link: [
+      {
+        rel: "icon",
+        type: "image/x-icon",
+        href: "/img/12-labours-logo-black.png",
+      },
+    ],
   },
-  env: {
-    graphcms_api: process.env.GRAPHCMS_ENDPOINT,
-    google_analytics_ga4:process.env.GOOGLE_ANALYTICS_GA4,
+
+  publicRuntimeConfig: {
+    portal_url: process.env.PORTAL_URL || "http://localhost:3000",
+    query_api_url: process.env.QUERY_API_URL || "http://localhost:8000",
+    login_api_url: process.env.LOGIN_API_URL || "http://localhost:8080",
+    login_api_key: process.env.LOGIN_API_KEY,
+    login_secret_key: process.env.LOGIN_SECRET_KEY,
+    graphcms_endpoint: process.env.GRAPHCMS_ENDPOINT,
+    google_client_id: process.env.GOOGLE_CLIENT_ID,
+    twelve_labours_xml: process.env.TWELVE_LABOURS_XML,
+    google_analytics_ga4: process.env.GOOGLE_ANALYTICS_GA4,
+    flatmap_api: process.env.FLATMAP_API,
+    axios: {
+      baseURL: process.env.LOGIN_API_URL || "http://localhost:8080",
+    },
     social_twitter:

```

Comment:
Some of these should only be exposed to privateRuntimeConfig.

Reply:
By the way, quoted from this page here https://v2.nuxt.com/docs/directory-structure/nuxt-config/: 
`Your private config could be exposed if you use $config outside of a server-only context (for example, if you use $config in fetch, asyncData or directly inside your template).`